website/docs/r/vapp.html.markdown

```diff
@@ -24,12 +24,6 @@ resource "vcd_network_direct" "net" {
 resource "vcd_vapp" "web" {
   name = "web"
 
-  metadata {
```

Comment:
First I thought why this has been deleted, but then I realised that you removed what is not working yet (until @dn1s metadata PRs are merged). The doc for the old way of defining metadata in a vApp with VM in a _single_ resource (which we want to deprecate and which is described in the doc below) has been left.